core/rawdb/freezer_table.go

```diff
@@ -106,6 +107,44 @@ func newTable(path string, name string, readMeter metrics.Meter, writeMeter metr
 	return newCustomTable(path, name, readMeter, writeMeter, 2*1000*1000*1000, disableSnappy)
 }
 
+// openFreezerFile opens a freezer table file and seeks to the end
+func openFreezerFileForAppend(filename string) (*os.File, error) {
```

Comment:
Nitpick, please fix the comments so they reference the correct method (on this and the 2 methods below).

Reply:
And in general, please leave an empty space after the `//` characters within the method comments.

Reply:
And please capitalize min-method comments :)

Reply:
lol, well spotted. thanks, and done :-)